xbmc/video/VideoDatabase.h

```diff
@@ -83,6 +83,9 @@ namespace VIDEO
 #define VIDEODB_DETAILS_MOVIE_DATEADDED         VIDEODB_MAX_COLUMNS + 10
 #define VIDEODB_DETAILS_MOVIE_RESUME_TIME       VIDEODB_MAX_COLUMNS + 11
 #define VIDEODB_DETAILS_MOVIE_TOTAL_TIME        VIDEODB_MAX_COLUMNS + 12
+#define VIDEODB_DETAILS_MOVIE_RATING            VIDEODB_MAX_COLUMNS + 13
+#define VIDEODB_DETAILS_MOVIE_VOTES             VIDEODB_MAX_COLUMNS + 14
+#define VIDEODB_DETAILS_MOVIE_DEFAULT_RATING    VIDEODB_MAX_COLUMNS + 15
```

Comment:
The column name is `rating_type` which doesn't match `DEFAULT_RATING` unless I misunderstood something.